scripts/postcss/README.md

```diff
@@ -0,0 +1,99 @@
+# Terra Postcss Theme plugin
+
+The purpose of this plugin is to create a default theme from a scoped theme and to remove any known themes that are not desired.
+
+The plugin will read from the same terra-theme.config.js that terra-aggregate-themes script reads from. Eventually terra-aggregate-themes will be deprecated in favor of this strategy.
+
+## Configuration
+
+### terra-theme.config.js
+
+Below is an example of terra-theme.config
+
+```js
+const themeConfig = {
+  exclude: [], // Files to exclude. Accepts glob patterns.
+  theme: 'terra-dark-theme', // The default theme.
+  scoped: ['terra-light-theme', 'terra-lowlight-theme'], // An array of scoped themes.
```

Comment:
It might be helpful to explain the difference/use-case of theme vs scoped or link the original docs

Reply:
I'll add a stand alone theme config doc, but i'm not sure how i want to link theme yet.